src/config/app-config.interface.ts

```diff
@@ -25,6 +25,9 @@ import { FilterVocabularyConfig } from './filter-vocabulary-config';
 interface AppConfig extends Config {
   ui: UIServerConfig;
   rest: ServerConfig;
+  ssr: {
```

Comment:
`BuildConfig` may be a more appropriate place for this

Reply:
I have no strong preference. @tdonohue where would you prefer?

Reply:
I think we should find a way to get this fully working before we tweak it this PR much further.  Minor tweaks here don't matter too much until we can a solution we like. 

Arguably, these settings could also move entirely under the existing `universal` configs, because that's where other settings related to SSR currently reside.  So, we could restructure this like:

```
universal:
  preboot: true
  async: true
  time: false
  rest:
     ssl: true
     host: api7.dspace.org
     port: 443
     nameSpace: /server
```

That would clarify that these settings are directly related, rather than having different config sections for "universal" vs "ssr".